prompt/ykman.go

```diff
@@ -0,0 +1,16 @@
+package prompt
+
+import (
+	"os/exec"
+	"strings"
+)
+
+func YkmanPrompt(_prompt string, mfa_serial string) (string, error) {
```

Comment:
`_prompt`?

Reply:
I believe that's the standard for "unused, but I wanted to say what it is anyway" naming? Happy to change it to just 
`_` if you prefer.